app/views/docs/rest.phtml

```diff
@@ -1,12 +1,310 @@
 <p>Appwrite supports multiple protocols for accessing the server, including <a href="/docs/rest">REST</a>, <a href="/docs/graphql">GraphQL</a>, and <a href="/docs/realtime">Realtime</a>.</p>
 
-<p>The REST API allows you to manipulate your Appwrite server through many endpoints, each representing a specific method of modification to a specific resource type.</p>
+<p>The REST API allows you to access your Appwrite server through HTTP requests, without the need for an SDK. Each endpoint in the API represents a specific operation on a specific resource.</p>
 
-<p>Requests and responses through the REST API follow a rigid structure - that is, they always expect the same structure for requests to a specific endpoint, and responses from that endpoint will also always have the same structure. </p>
+<h2><a href="/docs/rest#headers" id="headers">Headers</a></h2>
+<p>Appwrite's REST APIs expect certain headers to be attached to each request:</p>
 
-<p>The specific structures used for Appwrite resources and their related endpoints can be found in the <b>References</b> section for each service.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 300px">Header</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>X-Appwrite-Project: [PROJECT-ID]</code></td>
+        <td>Required, provides the ID of your Appwrite Project to the REST API.</td>
```

Comment:
It might be useful to also document somewhere that you can also pass the project ID in the URL via a query string like:

```
?project=[PROJECT-ID]
```

Reply:
Ohhhh that I did not know! Is this ever a preferred practice? Maybe for public image previews?

Reply:
Exactly, this is required for images in HTML because you can't include headers for the `<img>` element:

```html
<img alt="image" src="penguin.jpg" />
```